test/e2e/specs/all-features-confirmation/record/presentation.spec.js

```diff
@@ -114,16 +114,20 @@ describe('View existing record,', function() {
 
             browser.wait(function() {
                 return chaisePage.recordPage.getRelatedTablesWithPanel().count().then(function(ct) {
+                    console.log("ct: "+ct);
                     return (ct=testParams.no_related_data.tables_order.length);
                 });
             }, browser.params.defaultTimeout);
             var showAllRTButton = chaisePage.recordPage.getShowAllRelatedEntitiesButton();
 
             chaisePage.recordPage.getRelatedTablesWithPanel().count().then(function(count) {
+                browser.sleep(2000);
```

Comment:
Maybe wait for the `loading...` element to not be visible anymore? That element should show up at the very bottom of the page and disappear once all related tables are loaded. But this should be done before you even try to get their count. a sleep here shouldn't make any difference at all since it's in the promise after the value was already fetched.